api/v1/application/repository_latest.go

```diff
@@ -106,8 +107,21 @@ func getWhereClause(criteria map[string]interface{}) string {
 					OR cast("av".manifest as TEXT) ILIKE '%%%s%%'
 					OR cast("av".tags as TEXT) ILIKE '%%%s%%'
 				) `, value, value, value, value, value, value)
+		} else if strings.Contains(key, ".") {
```

Comment:
Cyclomatic and N-path complexities are too high:
```
if {
} else if {
  for {
   if {
   } elseif {
   } else {
   }
  }
} else {
}
```

This should be refactored with submethods to keep only one level, avoiding `else if` (and `else`, as much as possible).